logic/common/file_utils.py

```diff
@@ -130,4 +131,30 @@ def StripFilename(file_path):
     # Split the filename and the extension
     filename_without_extension, _ = os.path.splitext(filename_with_extension)
 
-    return filename_without_extension
\ No newline at end of file
+    return filename_without_extension
+
+
+
+
+
+def GetFile(filename):
```

Comment:
'GetFile' doesn't work as a name since this function is not creating a file.

The way I described it in trello, GetFile would have ensured the folder exists as well as create the file - then return the file handler.

But you have that step done outside. Which I think is fine. Can just rename this function to "EnsureFolderExists"